.github/workflows/complete-e2e-tests.yml

```
@@ -1,12 +1,12 @@
 # Builds JPlag and then runs Datasets and Report Viewer e2e tests on multiple OS
-name: Complete e2e Test
+name: Test Complete Usage Workflow
```

```suggestion
name: Complete E2E Tests
```

This was still the name we proposed, but since End-to-end tests are a common concept, I say Timur's suggestion is also fine – with the proper capitalization.

---

looking for better name

---

Name it just `End-to-End Tests` name file `test-e2e.yml`
Rename core e2e test package to something else